src/lib/reasoners/satml_frontend.ml

```diff
@@ -539,26 +539,6 @@ module Make (Th : Theory.S) : Sat_solver_sig.S = struct
          with Not_found -> inst, acc
       ) (env.inst, acc) sa
 
-  let measure at =
-    Atom.level  at,
-    Atom.weight at,
-    Atom.index  at
-
-  (* smaller is more important *)
-  let cmp_tuples (l1, w1, i1) (l2,w2, i2) =
-    (* lower decision level is better *)
-    let res = compare l1 l2 in
-    if res <> 0 then res
-    else
-      (* higher weight is better hence compare w2 w1 *)
-      let res = Stdlib.compare w2 w1 in
-      if res <> 0 then res
-      else
-        (* lower index is better *)
-        compare i1 i2
-
-  let max a b = if cmp_tuples a b > 0 then a else b
-
```

Comment:
Please, can you explain what was the purpose of this code and why we don't need it anymore?

Reply:
I have no idea what was the purpose of this code but it is only used to compute some value in `atoms_from_sat_branch` which is then ignored (see `Some (_, l) -> ...` line 628).